waspc/data/Generator/templates/Dockerfile

```diff
@@ -25,16 +25,15 @@ RUN apk add --no-cache openssl1.1-compat
 FROM base AS server-builder
 RUN apk add --no-cache build-base libtool autoconf automake
 WORKDIR /app
+COPY server/ ./server/
 # Install npm packages, resulting in node_modules/.
-{=# usingServerPatches =}
-COPY server/patches ./server/patches
-{=/ usingServerPatches =}
-COPY server/package*.json ./server/
 RUN cd server && npm install
 {=# usingPrisma =}
 COPY db/schema.prisma ./db/
 RUN cd server && {= serverPrismaClientOutputDirEnv =} npx prisma generate --schema='{= dbSchemaFileFromServerDir =}'
 {=/ usingPrisma =}
+# Compile TS (depends on Prisma, if in use)
```

Comment:
Nit: This comment feels a bit off

The fact that we are compiling TS here, it's not really important, we are building the server in general. If I understood the comment correctly, you wanted to convey the idea "we need to put this AFTER the `prisma generate` bit". Maybe write it like that 🤷

Reply:
Good point, I can drop the compile/build note and just say we need to do this after Prisma entities were generated. 👍🏻 Will update comment shortly